frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/MfaResetAuthorizeHandlerTest.java

```diff
@@ -0,0 +1,143 @@
+
+import java.util.Map;
+import java.util.Optional;
+
+import static java.lang.String.format;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.di.authentication.frontendapi.helpers.CommonTestVariables.CLIENT_SESSION_ID;
+import static uk.gov.di.authentication.frontendapi.helpers.CommonTestVariables.DI_PERSISTENT_SESSION_ID;
+import static uk.gov.di.authentication.frontendapi.helpers.CommonTestVariables.EMAIL;
+import static uk.gov.di.authentication.frontendapi.helpers.CommonTestVariables.ENCODED_DEVICE_DETAILS;
+import static uk.gov.di.authentication.frontendapi.helpers.CommonTestVariables.INTERNAL_SUBJECT_ID;
+import static uk.gov.di.authentication.frontendapi.helpers.CommonTestVariables.IP_ADDRESS;
+import static uk.gov.di.authentication.frontendapi.helpers.CommonTestVariables.SESSION_ID;
+
+class MfaResetAuthorizeHandlerTest {
+    private final ConfigurationService configurationService = mock(ConfigurationService.class);
+    private final MfaResetIPVAuthorizationService mfaResetIPVAuthorizationService =
+            mock(MfaResetIPVAuthorizationService.class);
+    private final AuthenticationService authenticationService = mock(AuthenticationService.class);
+    private final ClientSessionService clientSessionService = mock(ClientSessionService.class);
+    private final ClientService clientService = mock(ClientService.class);
+    private final Context context = mock(Context.class);
+    private final SessionService sessionService = mock(SessionService.class);
+    private final UserContext userContext = mock(UserContext.class);
+    private final Session session = mock(Session.class);
+    private final AuditContext testAuditContext =
+            new AuditContext(
+                    AuditService.UNKNOWN,
+                    CLIENT_SESSION_ID,
+                    SESSION_ID,
+                    AuditService.UNKNOWN,
+                    EMAIL,
+                    IP_ADDRESS,
+                    AuditService.UNKNOWN,
+                    DI_PERSISTENT_SESSION_ID,
+                    Optional.of(ENCODED_DEVICE_DETAILS));
+    private final APIGatewayProxyRequestEvent TEST_INVOKE_EVENT =
+            ApiGatewayProxyRequestHelper.apiRequestEventWithHeadersAndBody(
+                    CommonTestVariables.VALID_HEADERS,
+                    format("{ \"email\": \"%s\" }", EMAIL.toUpperCase()));
+    private MfaResetAuthorizeHandler handler;
+
+    @BeforeEach
+    void testSetup() {
+        mock(SessionService.class);
+        mock(KmsConnectionService.class);
+        mock(JwtService.class);
+        mock(TokenService.class);
+        mock(CloudwatchMetricsService.class);
+        mock(AuditService.class);
```

Comment:
I don't think this is doing what you think it is

Reply:
Yep have deleted all of these as they're redundant